contrib/verify/verify.sh

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bash
+#
+# Script to verify all Tetragon programs by loading them with bpftool.
+#
+set -u
+shopt -s nullglob
+
+RED="\033[31m"
+BLUEUNDER="\033[34;4m"
+GREEN="\033[32m"
+NOCOLOR="\033[0m"
+TETRAGONDIR=/var/lib/tetragon
+DEBUG=0
+
+usage() {
+	echo "usage: verify.sh [-d] [TETRAGONDIR]"
+	echo "-d to run bpftool with -d and dump number of processed instructions"
+	exit 1
+}
+
+if [ $# -ge 1 ]; then
+	if [ "$1" == "-d" ]; then
+		DEBUG=1
+		shift 1
+	fi
+
+	if [ $# -ge 1 ]; then
+		[ -d $1 ] || usage
+		TETRAGONDIR=$1
+	fi
+fi
+
+PINDIR=/sys/fs/bpf/tetragon-verify
+mkdir -p "$PINDIR"
+cleanup() {
+	rm -rf "$PINDIR"
+}
+trap cleanup EXIT
+
+STATUS=0
+
+for obj in "$TETRAGONDIR"/*.o; do
+	B=$(basename "$obj")
+
+    # Alignchecker is not a bpf program, so ignore it
+	if [[ "$B" == bpf_alignchecker* ]]; then
+		continue
+	fi
+
+    # Globals is just for testing, so ignore it
+	if [[ "$B" == bpf_globals* ]]; then
+		continue
+	fi
+
+    # Generic tracepoint needs more complex userspace logic to load, so ignore it
```

Comment:
Might be worth adding an issue for this.

Reply:
Maybe so, although I'm not 100% sure I fully understand the specifics of the problem. The original script ignored _all_ `bpf_generic*` object files. I tried removing it but noted that the generic tracepoint code would fail to load. So I made the exception more specific and added a comment.

Haven't had a chance to actually investigate _why_ bpftool can't load these programs, but I assume our userspace loader logic is doing something clever with them before load time to make them work.

Reply:
Maybe @joamaki could provide some context behind the original script?

Reply:
My guess is that they need the event_config map, but could be mistaken.

In any case, I think something simple such as: "fix the verifier script to load bpf_generic* files" should give enough of a context for the person reading the issue to investigate.

Reply:
Sounds good, I'll create one when we land this

Reply:
made this: https://github.com/cilium/tetragon/issues/229.

I think it's better :)